pycqed/measurement/multi_qubit_module.py

```diff
@@ -626,10 +626,10 @@ def find_optimal_weights(dev, qubits, states=('g', 'e'), upload=True,
         unique, counts = np.unique(uhf_names, return_counts=True)
         for u, c in zip(unique, counts):
             if c != 1:
-                raise ValueError(f"{np.array(qubits)[uhf_names == u]}"
-                                 f" share the same UHF ({u}) and therefore"
-                                 f" their timetraces cannot be computed "
-                                 f"simultaneously.")
+                log.warning(f"{np.array(qubits)[uhf_names == u]} "
+                            f"share the same UHF ({u}) and therefore their "
+                            f"timetraces should not be measured simultaneously, "
+                            f"except you know what you are doing.")
```

Comment:
"except if you know what you are doing."

Reply:
Someone needed it for debugging RO. I would guess that it was related to RO crosstalk/mixing between qubits on a flux line. If you weren't the one who needed it, then maybe @nathlacroix knows more?

Reply:
That is correct @chellings , we used it with Ants when debugging TWPA/RO  crosstalk. Perhaps we could change the comment to "... share the same UHF ({u}) and in a standard multiplexed readout setting those should be calibrated separately."